cosmo/monitors/dark_monitors.py

```diff
@@ -0,0 +1,73 @@
+from typing import Any
+
+import numpy as np
+import plotly.graph_objs as go
+import pandas as pd
+
+from itertools import repeat
+
+from monitorframe.monitor import BaseMonitor
+from .data_models import FUVDarkDataModel
+from ..monitor_helpers import explode_df, absolute_time
+
+
+def dark_filter(df_row, filter_pha, location):
+    good_pha = (2, 23)
+    time_step = 25
+    time_bins = df_row['TIME_3'][::time_step]
+    lat = df_row['LATITUDE'][::time_step][:-1]
+    lon = df_row['LONGITUDE'][::time_step][:-1]
+    event_df = df_row[['SEGMENT', 'XCORR', 'YCORR', 'PHA', 'TIME']].to_frame().T
+    event_df = explode_df(event_df, ['XCORR', 'YCORR', 'PHA', 'TIME'])
+    npix = (location[1] - location[0]) * (location[3] - location[2])
+    index = np.where((event_df['SEGMENT'] == 'FUVA') &
```

Comment:
This is where I think the 0's were coming from in the test plots. In `DarkDataModel`, you don't specify a particular segment, so there'd be both FUVA and FUVB darks included in `new_data`. Here, you filter on `segment == "FUVA"`, so that would result in empty arrays when an FUVB "row" was being filtered. I'm not entirely sure (haven't looked tooooooo hard) why that wouldn't cause an error somewhere, but my guess is that one of the other operations on that array results in 0 somehow. Also, since you're filtering on the segment in `get_data` now, there's no need to include the same filter here.